cdap-client-tests/src/test/java/io/cdap/cdap/client/WorkflowClientTestRun.java

```diff
@@ -70,10 +72,15 @@ public void setUp() throws Throwable {
 
   @After
   public void tearDown() throws Exception {
-    appClient.delete(appId);
+    ApplicationDetail appDetail = appClient.get(appId);
+    appClient.delete(new ApplicationId(appId.getNamespace(), appId.getApplication(), appDetail.getAppVersion()));
```

Comment:
Some here. We should not need to provide a version. Delete will delete all versions

Reply:
so this test deletes a specifc version - the delete method calls the /version api

Reply:
This is teardown - meaning cleanup. Should not cleanup just remove all?

Reply:
there is only one version of the app created - if we want to change the delete function to not use the /version endpoint, we can replace that